monkey/infection_monkey/monkey.py

```diff
@@ -256,8 +261,10 @@ def self_delete():
                                      close_fds=True, startupinfo=startupinfo)
```

Comment:
What should I report here if we can't be sure the delayed process deletes the file?